src/components/ContentWrapper.jsx

```diff
@@ -96,15 +86,12 @@ const ContentWrapper = (props) => {
 
       io.on(`ExperimentUpdates-${experimentId}`, (update) => cb(experimentId, update));
     })();
-  }, [experimentId]);
-
-  useEffect(() => {
-    if (backendStatusRequested) {
-      return;
+    if ((![RUNNING, SUCCEEDED].includes(gem2sStatusKey)
```

Comment:
Checking whether we can re run or not gem2s or the qc pipeline is handled in the api, I don't think we need to add more checks here too, just to keep it simple.

Reply:
I removed those checks because its much faster and i think better to only call the api when needed. If the checks are done in the api it is delayed a little bit so thats why the user is in data-processing first and then gem2s is triggered again

Reply:
@kafkasl what do you think about this? I am not sure whether the check for whether gem2s should be rerun should be done in the api or the ui